index.html

```diff
@@ -1447,7 +1448,7 @@ <h4>Listing</h4>
 						</p>
 							
 						<p>
-							For pagination information the alternative format leans against the formats as described
+							For pagination information the alternative format leans on the formats as described
```

Comment:
"leans on" --> "depends upon" or "based upon".  However "based" is not quite right, these formats are "derived from" Hydra.  My preference would be "derived from" (and "described in").